P5/Test/expected-results/detest_odd_schematron.log

```diff
@@ -1,11 +1,11 @@
-Buildfile: /TEI/P5/Test/antruntest.xml
+Buildfile: /Users/syd/Documents/TEI_gitHub/P5/Test/antruntest.xml
```

Comment:
I presume that the expected results here should not be user specific -- should these all be `/TEI/` ?